.dockerignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
Consider adding more entries to the `.dockerignore` file to exclude unnecessary files and directories from the Docker image. Common entries include `.git`, `__pycache__`, and any local configuration files.